GitHub Action error: "bash: mypy: command not found"

I have stumbled upon this problem with my project https://github.com/python-platonic/documented.

Let me quote the piece of `.github/workflows/test.yml`, annotated with the printout.


```yaml
    - name: Run checks
      run: |
        source "$HOME/.poetry/env"

        find "$HOME" -name "mypy" -print
        # /home/runner/work/documented/documented/.venv/lib/python3.8/site-packages/mypy
        # /home/runner/work/documented/documented/.venv/bin/mypy

        echo $PATH
        # /home/runner/.poetry/bin:/opt/hostedtoolcache/Python/3.8.5/x64/bin:/opt/hostedtoolcache/Python/3.8.5/x64:/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:/usr/share/rust/.cargo/bin:/home/runner/.config/composer/vendor/bin:/home/runner/.dotnet/tools:/snap/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin

        make test
        # mypy documented tests/*.py
        # bash: mypy: command not found
        # make: *** [lint] Error 127
```

`mypy` command is in `/home/runner/work/documented/documented/.venv/bin/`, which is not in `PATH`.

--- a/{{cookiecutter.project_name}}/.github/workflows/test.yml
+++ b/{{cookiecutter.project_name}}/.github/workflows/test.yml
@@ -35,11 +35,12 @@ jobs:
     - name: Run checks
       run: |
         source "$HOME/.poetry/env"
+        export PATH="$(pwd)/.venv/bin:$PATH"
         make test
-        
+
     # Uncomment these lines if you wish to upload coverage to codecov:
     # https://codecov.io/
-    # - name: Upload coverage to Codecov  
+    # - name: Upload coverage to Codecov
     #   uses: codecov/codecov-action@v1
     #   with:
     #     file: ./coverage.xml

